src/perl/lib/wtsi_clarity/util/clarity_elements.pm

```diff
@@ -26,14 +28,14 @@ sub _set_udf_element {
 ## no critic(Subroutines::ProhibitManyArgs)
 sub _set_any_element {
   my ($self, $xml, $name, $value, $override, $is_udf) = @_;
-## use critic
+  ## use critic
```

Comment:
We need to refactor this to use less parameters, so we would get rid of the 'no critic' tag